mysql-test/mysql-test-run.pl

```diff
@@ -4509,7 +4509,7 @@ ($$)
      qr/Detected table cache mutex contention at instance .* waits. Additional table cache instance cannot be activated: consider raising table_open_cache_instances. Number of active instances/,
      qr/WSREP: Failed to guess base node address/,
      qr/WSREP: Guessing address for incoming client/,
-
+     qr/InnoDB: Difficult to find free blocks in the buffer pool*/,
      # for UBSAN
```

Comment:
I assume that there was a reason for that empty line to exist, to highlight the end of a "section". I would not remove the empty line, but just add the suppression right before it.